README.md

```diff
@@ -1,2 +1,243 @@
+
++ Next to Regents Canal (_perfect for people who cycle or like running_)
+
++ 4 Parks within walking distance.
+
++ Great Gym (2 minutes walk)
+
++ Out-door/Park Gym (_free. also 2 mins walk_)
+
++ Mile End Climbing Wall (5 minutes walk): https://www.mileendwall.org.uk/
+
++ Great restaurants & cafes nearby 5mins walk.
+
++ Free tea & coffee (_tell us what your favourite brand/flavour is and will make sure it's always in stock!_)
+
++ Free Fresh Fruit for members.
+
++ Blender and Juicer equipment so you can make your own. (_or for "cost price" you can buy made-this-moring cold-pressed organic juice or smoothies from our supplier_)
+
++ Fridge and microwave oven.
+
+#### Desk Rental Rates >> _Simple/Transparent Pricing_
+
+We aim to be _cost-effective_ while giving our members lots of added value.
+
++ Monthly Subscription **£180/desk** (_hot desk + clean desk policy_)
++ Pay-as-you-Go **£11/day** (_sold in blocks of 5 "credits"_)
+
+> "Old" Pricing: http://www.founderlondon.com/images/workhere.jpg <br />
+> Discussion: https://github.com/foundersandcoders/london-programme/issues/42
+
+### World-Class Meetup & Training Venue
+
+Part dwyl's of our mission to help everyone discover what they love doing requires us to share our knowledge through Meetups and training.
+
+We have _already **successfully**_ used the space for training both
+our own team members, our clients and complete strangers.
+
+Our plan is to create a _public_ training calendar for QuietSpace
+where specific training is regular.
+
+#### What Will We Learn?
+
+We already have a [52+](https://github.com/search?q=org%3Adwyl+learn)
+tutorials in various levels of "maturity" which we plan to convert into video content by recording a "complete beginners workshop"
+
+We have a _rough **order**_ for the learning material:
+https://github.com/dwyl/the-book/issues/69 but we expect to be "_dynamic_" and will respond to the requests/needs of the attendees/members.
+
+
+
+
+#### Training Registration System
+
+For the QuiteSpace-based training we will need a
+registration system/process so people can
+book a space on the training for a specific day/time.
+(_The **MVP** can use an
+**existing "booking" tool** provided we capture all the requirements
+before making a decision. as always we **prefer open source**._)
+
+
+
+
+### QuietSpace Timetable for Co-Working, Breaks and Meetups/Training
+
+The most productive people in the world all have a well-defined Timetable for getting their work done.
+
+|Start - End    | Description |
+|---------------|-------|
+| 04:00 - 06:00 | Healthy food preparation (_upstairs available for work, expect some noise will be made downstairs_) |
+| 06:00 - 07:00 | Self-guided Yoga / Pilates / Mindfulness (Downstairs), Reading or Quiet work Upstairs |
+| 07:00 - 10:00 | "Early Bird's" Focus Time (_absolute silence please this is when we get our best work done_) |
```

Comment:
Ines, this is self-directed yoga/pilates following a YouTube video the way I have been doing.
If Emma is _able_ to come to the space that's _awesome_. but we cannot put "Emma's Yoga" in the timetable as a "thing" because then people _Expect_ it and complain when Emma is no longer available.

_again_ this is a proposal. if you have a _better_ suggestion, please just update the doc directly. thanks! ❤️